README.md

```diff
@@ -353,33 +415,73 @@ We have to create a constructor in the `Human` class that takes a `Dog` instance
 as an argument and assigns it to the `dog` field in the class. Then we annotate
 the constructor with `@Autowired`.
 
+The most common way to use the `@Autowired` annotation is to create a constructor
+in the class that will reference another bean. In our case, we'll create an
+autowired constructor in the `Human` class that takes a `Dog` instance as an
+argument.
+
 ```java
 @Component
 public class Human {
     private String name;
-    private final Dog dog;
+    private Dog dog;
 
     @Autowired
     public Human(Dog dog) {
-        this.dog = dog;
+       
     }
+    
+    // getters and setters
+}
+```
+
+Since we are using the `@Autowired` annotation, we don't have to instantiate the
+`dog` field in the `Human` class - we can just assign it to the
+`Dog` argument passed into the constructor:
 
-		// getters, setters, and toString
-		// no setter for "dog" since it's a final field
+```java
+@Component
+public class Human {
+    private String name;
+    private Dog dog;
+
+    @Autowired
+    public Human(Dog dog) {
+       this.dog = dog;
+    }
+    
+    // getters and setters
+}
+```
+
+We can now also make the `dog` instance variable `final` as well, since the value
+is injected during object creation.
+
+```java
+@Component
+public class Human {
+    private String name;
+    private final Dog dog;
+
+    @Autowired
+    public Human(Dog dog) {
+       this.dog = dog;
+    }
+    
+    // getters and setters
+    // no setter for the dog since it is now a final field
```

Comment:
The README.md shows MyConfig as :  

package config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@Configuration
@ComponentScan(basePackages = "main")
public class MyConfig {

}